File: backend/src/main/java/com/daedan/festabook/place/domain/PlaceCategory.java
package com.daedan.festabook.place.domain;

public enum PlaceCategory {
    BOOTH,
    BAR,
    FOOD_TRUCK,
    SMOKING,
    TRASH_CAN;

    public boolean isServiceLocation() {
        return this == BOOTH || this == BAR || this == FOOD_TRUCK;
    }
}

File: android/app/src/main/java/com/daedan/festabook/domain/model/PlaceCategory.kt
package com.daedan.festabook.domain.model

enum class PlaceCategory {
    FOOD_TRUCK,
    BOOTH,
    BAR,
    TRASH_CAN,
    TOILET,
    SMOKING_AREA,
    PARKING,
    PRIMARY,
    STAGE,
    PHOTO_BOOTH,
    EXTRA
    ;

    companion object {
        val SECONDARY_CATEGORIES =
            listOf(
                TRASH_CAN,
                TOILET,
                SMOKING_AREA,
                PARKING,
                PRIMARY,
                STAGE,
                PHOTO_BOOTH,
                EXTRA,
            )
    }
}
